content/sensu-go/6.2/operations/monitoring-as-code/_index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Monitoring as Code with Sensu"
+linkTitle: "Monitoring as Code"
+description: "Read this page to learn about following a monitoring-as-code approach with Sensu."
+product: "Sensu Go"
+version: "6.2"
+weight: 5
+layout: "single"
+toc: true
+menu:
+  sensu-go-6.2:
+    parent: operations
+    identifier: monitoring-as-code
+---
+
+Sensu supports end-to-end monitoring as code so you can manage your observability and monitoring workflows the same way you build, test, and deploy your applications and infrastructure.
+Sensu allows you to define your entire monitoring workflow as declarative YAML or JSON code in configuration files, from collecting observability events and metrics and diagnosing issues through sending alerts and automatically remediating problems.
+
+In the monitoring as code approach, when a new endpoint starts up, like a cloud compute instance or Kubernetes Pod, Sensu automatically registers itself with the platform and starts collecting monitoring and observability data according to the code in your configuration files.
+If you manage your monitoring and observability workflow in the same way as the systems you're monitoring &mdash; as code, using a centralized continuous integration/continuous delivery (CI/CD) pipeline &mdash; you can align your monitoring with your product and services and improve visibility, reliability, and repeatability.
+
+- Your entire team can share, edit, review, and version Sensu configuration files just like they would with other "as-code" solutions.
+- Export your Sensu configuration from one environment and initialize another environment with the same configuration.
+- Use Sensu configuration files to remove, restore, back up, and recover Sensu instances.
+
+## Create a monitoring as code repository
+
+You can use any source control repository for maintaining your monitoring workflow configuration files.
+Your monitoring as code repository should include configuration files with the resource definitions you use in your monitoring workflow.
```

Comment:
I want to suggest what these configuration files might look like, but I'm not sure what we'd recommend.

For example, would we suggest putting all resources for a particular configuration into one file, like `test_resources.yaml`? Or would each file contain a type of resource, like `test_handlers.yaml`, `test_checks.yaml`, and so on?

Or would it be something else completely?